templates/taxonomy-topic.php

```diff
@@ -18,11 +20,15 @@
 		}
 
 		if ( have_posts() ) {
+			?>
+			<div class="<?php UCF_FAQ_Config::add_athena_attr( 'row' ); ?>">
+				<div class="<?php UCF_FAQ_Config::add_athena_attr( 'col-8' ); ?>">
```

Comment:
Consider updating these col classes to collapse at the -sm breakpoint (`.col-md-8`, `.col-md-4`) to accommodate the upcoming sidebar content